frontend/www/js/omegaup/components/course/StudentProgress.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <tr>
+    <td class="text-center align-middle">
+      <a v-bind:href="studentProgressUrl()">
+        {{ student.name || student.username }}
+      </a>
+    </td>
+    <td
+      class="score flex-column justify-content-center align-items-center"
+      v-for="assignment in assignments"
+    >
+      <p class="mb-1 text-center">{{ Math.round(score(assignment)) }}%</p>
+      <div class="d-flex justify-content-center">
+        <div v-if="student.progress.hasOwnProperty(assignment.alias) == false">
+          {{ T.wordsProblemsUnsolved }}
+        </div>
+        <div
+          v-for="problem in student.progress[assignment.alias]"
+          v-bind:class="getProblemColor(Math.round(problem))"
+          data-toggle="tooltip"
+          data-placement="bottom"
+          v-bind:title="problemScore(problem)"
```

Comment:
```suggestion
          v-bind:title="`{Math.round(problemScore)}%`"
```

this expression is simple enough that it's probably better to inline it.